packages/lavapack/src/runtime-template.js

```diff
@@ -31,7 +31,7 @@
     loadModuleData,
     getRelativeModuleId,
     prepareModuleInitializerArgs,
-    globalThisRefs: ['window', 'self', 'global', 'globalThis'],
+    globalThisRefs: ['window', 'self', 'global', 'globalThis', 'top', 'frames', 'parent'],
```

Comment:
is it possible to stuff this list somewhere where we don't have to copy/paste it to a bunch of places?

Reply:
well, if it exists in the source but not the tests, when the source changes, the tests won't reflect those changes.   so yeah, IMO it's worth it to share the data structure as much as is reasonable

Reply:
ok sure.
how would you got about @boneskull?
we don't have some shared space for such stuff, do we? should i just place this in `core`? and then import it from `browserify`? (these 2 cover all non-static occurrences)

Reply:
it seems reasonable to define some frozen objects in `core` and export them from e.g., `constants`. in this case, I'd probably want a "const enum"-type object:

```js
const DEFAULT_GLOBALTHIS_REFS = Object.freeze({
  WINDOW: 'window',
  SELF: 'self',
  GLOBAL: 'global',
  GLOBALTHIS: 'globalThis',
  TOP: 'top',
  FRAMES: 'frames',
  PARENT: 'parent',
});
```

This would probably be the most flexible data structure (easy conversion to array) and enables expressions like `DEFAULT_GLOBALTHIS_REFS.WINDOW !== globalThis.window` to mitigate bugs of the "typo" variety 😄

Reply:
how's this? https://github.com/LavaMoat/LavaMoat/pull/1539/files

Reply:
merged